app/templates/ui/app/user/user-dir.html

```diff
@@ -1,21 +1,15 @@
-<div class="pull-right user">
+<div class="user pull-right">
   <!-- form to log in -->
-  <form class="navbar-form navbar-right" ng-if="!ctrl.currentUser" ng-submit="ctrl.login()">
-    <div class="alert alert-danger" ng-if="ctrl.loginError" ng-cloak>Username and/or Password Incorrect</div>
-    <div class="form-group">
-      <input type="text" placeholder="Username" class="form-control input-sm" ng-model="ctrl.username">
-    </div>
-    <div class="form-group">
-      <input type="password" placeholder="Password" class="form-control input-sm" ng-model="ctrl.password">
-    </div>
-    <button type="submit" class="btn btn-success btn-sm">Sign in</button>
-  </form>
+  <div class="navbar-form navbar-right" ng-if="!ctrl.userService.currentUser()" >
+    <login ng-if="ctrl.loginService.loginMode() === 'top-right'" mode="inline"></login>
+    <span ng-if="ctrl.loginService.loginMode() !== 'top-right'" class="profile-link"><a ng-click="ctrl.loginService.loginPrompt()">Login</a></span>
```

Comment:
If you change this:
<a ng-click="ctrl.loginService.loginPrompt()">Login</a>

to this:

<a href ng-click="ctrl.loginService.loginPrompt()">Login</a>

then the cursor will change to the pointer cursor vs the text cursor w/o you having to set a style.